opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/export/__init__.py

```diff
@@ -519,7 +530,10 @@ def _shutdown():
             return
 
         self._shutdown_event.set()
-        self._daemon_thread.join(timeout=(deadline_ns - time_ns()) / 10**9)
+        if this._daemon_thread:
```

Comment:
This will give AttributeError because `this._daemon_thread` is only set when the condition evals true